webcompat/static/js/lib/bugform.js

```diff
@@ -159,10 +180,11 @@ function BugForm() {
       $("[value=" + problemType[1] + "]").click();
     }
 
-    // If we got a details param, add that to the end of the issue description.
+    // If we got a details param, add that to the end of the steps to reproduce field
+    // github.com/webcompat/webcompat.com/issues/1547#issuecomment-305649762
```

Comment:
Ah, yeah. I don't remember. As long as it does what we need it to do, I have no preference. :)